scripts/net/README.txt

```diff
@@ -0,0 +1,63 @@
+The shell script run-sample-tests.sh runs a few Zephyr samples against the
+network test applications container provided by the 'net-tools'
+Zephyr project, https://github.com/zephyrproject-rtos/net-tools.
+
+
+Installation
+************
+
+As a prerequisite it is assumed that Docker is installed and that the
+'net-tools' Docker container has been created, see the first bullet point
+at the net-tools Docker README file
+https://github.com/zephyrproject-rtos/net-tools/blob/master/README.docker.
+In essence, the following needs to be done:
+
+ * Install Docker
+ * Check out the net-tools project from github
+ * Change working directory to docker/ in the net-tools repository
+ * Run './net-setup.sh --config docker.conf'
```

Comment:
Maybe 'sudo ./net-setup.sh --config docker.conf'

Reply:
The script will do sudo if needed.